e2e/tests/ui/notebook-ui.spec.ts

```diff
@@ -19,17 +20,20 @@ describe('Notebook and Runtime UI Test', () => {
   test('Notebook name is unique', async () => {
     const existWorkspace = await openWorkspace(page, workspaceName);
     if (!existWorkspace) {
+      logger.info(`Cannot find workspace "${workspaceName}". Test end early.`);
       return;
```

Comment:
What's the correct behavior when a test precondition is not met: pass or fail?

Reply:
It's pass. Targeted workspace and notebook will be available after `notebook-create-python.spec` ran.